21/API/Service.swift

```diff
@@ -53,15 +53,15 @@ struct AuthService {
 
 static let shared = AuthService()
```

Comment:
Change the name to 'Service' or 'PokemonService'
AuthService doesn't make sense